### DISCUSSION TITLE
Definición de trabajo final tercer semestre

### DISCUSSION THREAD

**Discussion by SebasGalvan** [CodeSystem2022/Debugging_Demons_Tercer_Semestre#20] [Ideas]
## Definición TP tercer semestre Grupo **Debugging Demons**  🐱‍🏍

 En este post vamos a proponer ideas sobre el trabajo final del tercer semestre del Grupo **Debugging Demons** 😈
- Podemos mantener el mismo trabajo que el semestre pasado
- Podemos iniciar uno nuevo.

**Comment by BraianTroncoso** [comment]
Por mi parte me gustaría uno nuevo, alguna aplicación Web para solicitar turnos, por ejemplo en una barbería(depaso me vendría al pelo :smile: )
Tengo algo ya de código se se quiere reutilizar.

**Comment by TanitoCode** [comment]
Prefiero hacer uno nuevo tambien, podriamos ir tirando ideas, concuerdo con Braian en hacer una aplicacion Web y de paso que le sirva para su barberia, obviamente nos pagaria con cortes gratis de por vida.

**Reply by BraianTroncoso** [reply]
Mejor pensemos en otro proyecto, salgo perdiendo jaja

**Comment by francogenre** [comment]
Primero me sumo a la idea del Tano de que tengamos cortes gratis de por vida, y segundo está buena la idea de la app que sugiere Braian. También me parecería bueno hacer algo diferente.

**Comment by Aguppesce** [comment]
También estoy de acuerdo en hacer algo nuevo, lo de la barbería me parece buena idea!

**Comment by Marianoleonardofarias** [comment]
Me sumo a hacer algo nuevo!. Escuche algunas ideas que comentaron sobre IA que seria algo nuevo y revolucionario para el mercado.
